server/routes/ad.ts

```diff
@@ -460,8 +523,8 @@ export default class AdService {
         mustQueries.push({
           query_string: {
             fields: ['indices'],
-            default_operator: 'OR',
-            query: `*${indices.trim().split(' ').join('* *')}*`,
+            default_operator: 'AND',
+            query: `*${indices.trim().split('-').join('* *')}*`,
```

Comment:
Will it work just splitting by "-" ? Have you tested with " "

Reply:
Yes, this is another part of the bug that needed to be fixed. Note that the `search` field also splits by `-` in order to properly search for detectors with a `-` in them. A similar fix needs to be done for detectors that are using indices with `-` in them. See [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/272#discussion_r472553954) for more details.